### PR TITLE
Remove run as root required permissions

### PR DESCRIPTION
**Pull request by sarroutbi**
Right now, tang-operator deployment has been changed
so that tang container is not launched as root user.
This allows removing 'oc adm policy' that was
previously required for tang-operator to deploy
tang-container appropriately

Signed-off-by: Sergio Arroutbi <sarroutb@redhat.com>

### CODE REVIEW

**Comment by sarroutbi** [comment]
Hello @TomasTomecek. May I ask if there have been recent updates in the deployment of the machines that are used for checks?

Previously, no issues existed: https://github.com/RedHat-SP-Security/tests/pull/63/checks?check_run_id=4548562102

There are issues now when trying to retrieve information from minikube, which in the end exports a port with the service installed. Everything works ok when executing it locally.

So, basically, fails are related to "wget" to those ports. It seems that new firewall rules or similar have been installed, but I got no permissions to dump iptables, install firewall-cmd, etc.

Can you please help?

**Comment by TomasTomecek** [comment]
@sarroutbi yes, you can get access to the machine and "live debug" the problem. I suggest either opening an issue in the Testing Farm project https://gitlab.com/testing-farm/general/-/issues to get more details about what changes the team has done to the provisioning of the machines; or on the internal IRC `#tmt` channel to discuss

BTW I am very slow with processing of my GitHub notifications so I suggest opening issues (testing farm over there ^, for packit and the GitHub integration at https://github.com/packit/packit/issues/new) or reaching out on #packit libera IRC channel

Thanks!

**Comment by sarroutbi** [comment]
> 

Hello @TomasTomecek . Thank you very much for the instructions on how to debug this. I have opened a ticket through:
 https://gitlab.com/testing-farm/general/-/issues
 
 BTW, no problem at all in the slow processing, as this is not urget, and thanks for sharing a more direct way for reaching out.
